src/main/java/Command.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Command takes in the parsed input from the user and splits it into three categories.
+ * They are the command type, description of the task and the date. Depending on the
+ * type of command, description and date may be empty. Command then returns the
+ * appropriate mesage to return to Logic class to output to the user.
+ *
+ * @param "commandInput"  type of command.
+ * @param "descriptionInput" Y description of task.
+ * @param "dateInput" event date/deadline.
+ * @return output message to the user.
+ */
+public class Command extends Logic {
+    protected Boolean isBye;
+    protected String command;
+    protected String description;
+    protected String date;
+    protected String returnString;
+
+    public Command(String commandInput, String descriptionInput, String dateInput) {
+        this.command = commandInput;
+        this.description = descriptionInput;
+        this.date = dateInput;
+        this.isBye = false;
+    }
+
+    /**
+     * Executes the given command received from the Parser class.
+     * Returns the appropriate message to the Logic class.
+     *
+     * @return output message to the user.
+     */
+    public String executeCommand() {
+        switch (command) {
+        case "bye":
+            isBye = true;
+            returnString = Message.printExit();
+            break;
+        case "todo":
+            if (lIST_INDEX < LIST_LENGTH) {
```

Comment:
lIST_INDEX should probably be named LIST_INDEX, to avoid confusion between l (lowercase L) and I (uppercase I)
```suggestion
            if (LIST_INDEX < LIST_LENGTH) {
```

Reply:
Thanks for pointing that out I did not catch this typo!